lib/matplotlib/backends/backend_qt5agg.py

```diff
@@ -244,14 +190,11 @@ def __init__(self, figure):
         if DEBUG:
             print('FigureCanvasQtAgg: ', figure)
         super(FigureCanvasQTAgg, self).__init__(figure=figure)
-        self._drawRect = None
-        self.blitbox = []
```

Comment:
This going private needs to be documented and gracefully deprecated.  Users who use `blitting` are extra clever and are likely abusing this.